src/Raven.Studio/typescript/models/database/tasks/periodicBackup/s3Settings.ts

```diff
@@ -68,18 +82,41 @@ class s3Settings extends amazonSettings {
                 }
             ]
         });
+        
+        this.customServerUrl.extend({
+            required: {
+                onlyIf: () => this.useCustomS3Host()
+            },            
+            validation: [
+                {
+                    validator: (customServerUrl: string) => !this.useCustomS3Host() ||
```

Comment:
why can't we use validUrl validation rule?

Reply:
ok - using instead:

```
 validUrl: {
                onlyIf: () => this.useCustomS3Host()
 }
```

Reply:
do we need to use this with onlyIf - validurl doesn't compain when value is empty